api.md

```diff
@@ -670,6 +675,157 @@ Response
 }
 ```
 
+<a name=tailnet-keys></a>
+
+### Keys
+
+<a name=tailnet-keys-get></a>
+
+#### `GET /api/v2/tailnet/:tailnet/keys` - list the keys for a tailnet
+
+Returns a list of active keys for a tailnet.
+Supply the tailnet of interest in the path.
```

Comment:
It may be obvious, but should we callout that this list will be the user's active keys. (i.e. not all active keys in the whole tailnet)